samples/hello-android/settings.gradle.kts

```diff
@@ -1,7 +1,7 @@
 pluginManagement {
     repositories {
         gradlePluginPortal()
-        maven { url = uri("https://jcenter.bintray.com/") }
+        maven { url = uri("https://maven.google.com") }
```

Comment:
Can you not use `google()` here? or atleast `google().url`?

Reply:
No, not yet ;)